ir/src/ir.rs

```diff
@@ -11,6 +19,28 @@ pub struct Axis {
     pub hidden: bool,
 }
 
+pub type DesignSpaceLocation = HashMap<String, f32>;
+
+/// A variable definition of a single glyph.
+///
+/// Defined in at least once position in designspace. If defined in
+/// many, presumed to vary continuously between positions and required
+/// to have variation compatible structure.
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+pub struct Glyph {
+    pub name: String,
+    pub instances: Vec<GlyphInstance>,
+}
+
+/// A Glyph at a specific position in designspace.
+#[derive(Serialize, Deserialize, Debug, PartialEq)]
+pub struct GlyphInstance {
+    pub location: DesignSpaceLocation,
+    pub width: Option<f32>,
+    pub height: Option<f32>,
+    // TODO: outlines, a Vec<Shape>
```

Comment:
@anthrotype it seems per https://github.com/googlefonts/oxidize/pull/40 we should split contours and components.